### DISCUSSION TITLE
Add Custom Tool (UI)

### DISCUSSION THREAD

**Discussion by thomasjoubert** [strnad/CrewAI-Studio#12] [Ideas]
As a newbie, CrewAi-Studio allowed me to finally succeed in launching my first crew.
Unfortunately, I just broke the install by trying to add my own custom tool. (issue with import)
It'd be cool to be able to add custom tools directly in the UI.

**Comment by Wireman728** [comment]
I see this is an old post but I have a semi working custom tool importer, still a bit buggy.  If anyone else is working on this I am interested to see what you have.  I am working on getting what I have forked or branched but I am not sure how to do that.
